### PR TITLE
Testing output change

### PR DESCRIPTION
**Pull request by jmccreight**
There is a UPCOMING change to the wrfhydropy Simulation.output attribute which will break testing. These changes will ensure that wrf_hydro_nwm_public testing works on both sides of the changes to wrfhydropy.

### CODE REVIEW

File: tests/local/cheyenne/model_test.sh [hunk 1]
```diff
@@ -175,7 +176,8 @@ module list
 #-------------------------------------------------------
 # Python Env
 deactivate > /dev/null 2>&1
-source /glade/p/cisl/nwc/model_testing_env/wrf_hydro_nwm_test/bin/activate || exit 9
+# source /glade/p/cisl/nwc/model_testing_env/wrf_hydro_nwm_test/bin/activate || exit 9
+source /glade/work/jamesmcc/python_envs/36dv/bin/activate || exit 9
```
Comment (rcabell) [inline tests/local/cheyenne/model_test.sh]: Can this be moved into a /glade/p/cisl/nwc location instead of a personal directory?

Comment (jmccreight) [reply tests/local/cheyenne/model_test.sh]: 🤦‍♂ 
that was my intent, i really bungled this PR. I will fix it

Comment (jmccreight) [reply tests/local/cheyenne/model_test.sh]: um, yea, I did fix it. I belive you were looking at stale diffs

File: tests/local/cheyenne/model_test.sh [hunk 2]
```diff
@@ -176,6 +177,7 @@ module list
 # Python Env
 deactivate > /dev/null 2>&1
 source /glade/p/cisl/nwc/model_testing_env/wrf_hydro_nwm_test/bin/activate || exit 9
```
Comment (jmccreight) [inline tests/local/cheyenne/model_test.sh]: @rcabell

Comment (rcabell) [reply tests/local/cheyenne/model_test.sh]: Strange, I reloaded it this morning. Looks good now - thanks!

**Pull request merged**